lib/mocha.js

```diff
@@ -99,35 +97,14 @@ function Mocha(options) {
   // root suite
   this.suite = new exports.Suite('', new exports.Context(), true);
 
-  if ('useColors' in options) {
-    utils.deprecate(
-      'useColors is DEPRECATED and will be removed from a future version of Mocha. Instead, use the "color" option'
-    );
-    options.color = 'color' in options ? options.color : options.useColors;
-  }
-
-  // Globals are passed in as options.global, with options.globals for backward compatibility.
-  options.globals = options.global || options.globals || [];
-  delete options.global;
-
   this.grep(options.grep)
     .fgrep(options.fgrep)
     .ui(options.ui)
     .bail(options.bail)
-    .reporter(options.reporter, options.reporterOptions)
-    .useColors(options.color)
+    .reporter(options.reporter, options.reporterOption)
     .slow(options.slow)
     .useInlineDiffs(options.inlineDiffs)
-    .globals(options.globals);
-
-  if ('enableTimeouts' in options) {
-    utils.deprecate(
-      'enableTimeouts is DEPRECATED and will be removed from a future version of Mocha. Instead, use "timeout: false" to disable timeouts.'
-    );
-    if (options.enableTimeouts === false) {
-      this.timeout(0);
-    }
-  }
+    .globals(options.global);
```

Comment:
If we have removed the alias do you need to update it on https://github.com/mochajs/mocha/blob/master/docs/index.md#--global-variable-name ?

Reply:
No, the user can still use the alias `globals` **in the CLI**, the parser will understand the alias name and transform it to its canonical name `global`. The Mocha contructor processes only canonical names (or its camel-cased version), in order to avoid having several variables with the identical information which could loose synchronisation.